### PR TITLE
Adding markdown loader, readme page.

### PR DESCRIPTION
**Pull request by barrystaes**
The readme says markdown should be supported, this fixes that.
Also added page example that just shows README.md

### CODE REVIEW

File: app/Readme/index.jsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+	render: function() {
+		var style = {
+			default: {
+				"background-color": "white",
+				"border": "1px dotted gray",
+				"padding": "1em"
+			}
+		};
+		var readme = require("./../../README.md");
+		return <div style={style.default} dangerouslySetInnerHTML={{__html:readme}}></div>;
```
Comment (sokra) [inline app/Readme/index.jsx]: Could you change it to this?
``` js
		var readme = { __html: require("./../../README.md") };
		return <div style={style.default} dangerouslySetInnerHTML={readme}></div>;
```

File: app/Readme/index.jsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
+var React = require("react");
+
+var Readme = React.createClass({
+	render: function() {
+		var style = {
+			default: {
+				"background-color": "white",
```
Comment (sokra) [inline app/Readme/index.jsx]: I think it's `backgroundColor`.

**Comment by barrystaes** [comment]
Thanks.
I find the __html quite confusing, but thats apparently the intention with dangerouslySetInnerHTML. Insights welcome.

**Comment by sokra** [comment]
> PS. What is the argument for applying the __html key early on.. security, or keeping render() brief?

I think it intended to mark html strings with `{__html: "..."}` when passing them around. So you could find bugs when `dangerouslySetInnerHTML` is used with a normal string.

``` js
var a = {__html: "..."};
var b = getSomething(...);

<div dangerouslySetInnerHTML={condition ? a : b} />
```

Not obvious whether `getSomething` returns something html safe. By using the `{__html}` marker it'll tell use if it was just a string (which shouldn't be used because of security issues).

**Comment by barrystaes** [comment]
OK i see; this way searching for `__html` surfaces dangerous html usage at the source.

**Comment by barrystaes** [comment]
PS. In my inexperience i did this in my master branch, which of course is not practical for my development. If you could reject or accept this PR soon it would be much appreciated. :)

**Comment by sokra** [comment]
Sure

**Comment by sokra** [comment]
Thanks for the PR

**Pull request merged**